addons/survey/controllers/main.py

```diff
@@ -251,11 +252,17 @@ def submit(self, survey, **post):
     def print_survey(self, survey, token=None, **post):
         '''Display an survey in printable view; if <token> is set, it will
         grab the answers of the user_input_id that has <token>.'''
+        UserInput = request.env['survey.user_input']
+        user_input = UserInput.sudo().search([('token', '=', token)], limit=1)
+        test_entry = False
```

Comment:
If you don't have any user_input ```user_input.test_entry``` will be False
So this code could be expressed like this 
```test_entry = user_input.test_entry```
That will save up two lines